cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -0,0 +1,498 @@
+        json_cmd = json.loads(cmd)
+        json_cmd_str = json.dumps(json_cmd, sort_keys=True)
+        print("CMD: {}".format(json_cmd))
+        return self.return_val, \
+            self.cmd_output_map[json_cmd_str], \
+            "{}".format(self.err_message).encode('utf-8')
+
+    @classmethod
+    def Rados(conffile=None):
+        return DummyRados()
+
+
+try:
+    import rados
+except ModuleNotFoundError as noModErr:
+    # print("Error: %s\nExiting the script..." % noModErr)
+    # sys.exit(1)
+    rados = DummyRados
+
+try:
+    # for 2.7.x
+    from StringIO import StringIO
+except ModuleNotFoundError:
+    # for 3.x
+    from io import StringIO
+
+
+class ExecutionFailureException(Exception):
+    pass
+
+
+class RadosJSON:
+    EXTERNAL_USER_NAME = "client.healthchecker"
+    EMPTY_OUTPUT_LIST = "Empty output list"
+
+    @classmethod
+    def gen_arg_parser(cls, args_to_parse=None):
+        argP = argparse.ArgumentParser()
+        argP.add_argument("--verbose", "-v",
+                          action='store_true', default=False)
+        argP.add_argument("--ceph-conf", "-c",
+                          help="Provide a ceph conf file.", type=str)
+        argP.add_argument("--run-as-user", "-u",
+                          help="Provides a user name to check the cluster's health status",
+                          default=cls.EXTERNAL_USER_NAME, type=str)
+        argP.add_argument("--format", "-t", choices=["json", "bash", "shell"],
+                          default='shell', help="Provides the output format (json | shell)")
+        argP.add_argument("--cluster-name", default="openshift-storage",
+                          help="Ceph cluster name")
+        argP.add_argument("--output", "-o", default="",
+                          help="Output will be stored into the provided file")
+        argP.add_argument("--cephfs-filesystem-name", default="",
+                          help="Provides the name of the Ceph filesystem")
+        argP.add_argument("--cephfs-data-pool-name", default="",
+                          help="Provides the name of the cephfs data pool")
+        argP.add_argument("--rbd-data-pool-name", default="", required=True,
+                          help="Provides the name of the RBD datapool")
+        argP.add_argument("--namespace", default="",
+                          help="Namespace where CephCluster is running")
+        argP.add_argument("--rgw-pool-prefix", default="default",
+                          help="RGW Pool prefix")
+        argP.add_argument("--rgw-realm-name", default="",
+                          help="RGW Realm name")
+        argP.add_argument("--rgw-zonegroup-name", default="",
+                          help="RGW ZoneGroup name")
+        argP.add_argument("--rgw-zone-name", default="", help="RGW Zone name")
+        argP.add_argument("--rgw-endpoint", default="", required=True,
+                          help="Rados GateWay endpoint (in <IP>:<PORT> format)")
```

Comment:
These are not supported so let's remove them.

Reply:
So, shall we remove '--rgw-endpoint' as well?

Reply:
No.

Reply:
Clarified that `--rgw-endpoint` should not be removed...